components/cards/TransactionSummaryView.tsx

```diff
@@ -73,7 +74,21 @@ function TransactionSummaryView({
     paymentView = (
       <p>
         {isFairLaunch ? `Max Claim Payment (to Treasury): ` : "Claim Payment: "}
-        {truncateEth(ethers.utils.formatEther(claimPayment), 4)} {PAYMENT_TOKEN}
+        <InfoTooltip
+          content={
+            <div style={{ textAlign: "left" }}>
+              {isFairLaunch
+                ? "This is the amount you authorize for your claim payment. You'll only pay the Dutch auction price at the time of transaction confirmation."
+                : "Unclaimed parcels do not require a one-time payment after the conclusion of the Fair Launch Auction. Network fee payments still apply"}
```

Comment:
Add a period at the end of the second sentence in the claim payment tooltip.

 "Unclaimed parcels do not require a one-time payment after the conclusion of the Fair Launch Auction. Network fee payments still apply."

Reply:
Updated. Sorry about that. @gravenp